projects/commudle-admin/src/app/app.component.html

```diff
@@ -9,8 +9,10 @@
         <app-navbar-menu [currentUser]="currentUser"></app-navbar-menu>
 
         <div class="home-navbar-menu-user">
-          <nb-user *ngIf="currentUser" [nbContextMenu]="userContextMenu" class="menu-user-avatar" onlyPicture
-                   picture="{{currentUser.avatar}}"></nb-user>
+          <div class="menu-user-avatar">
```

Comment:
Move the *ngIf from the img tag to the surrounding div tag so that even the div tag doesn't get rendered if the user is logged out

Reply:
Okay